Flow Actions/Set Location/Set Field

```diff
@@ -0,0 +1 @@
+javascript: var luser = new GlideRecord("sys_user"); luser.get("sys_id", gs.getUserID()); luser.location.u_installation_id.sys_id.toString();
```

Comment:
This call type would not be suitable in a flow action. Additionally, you should use `getUniqueValue()` to get the sys_id string

Reply:
It works flawlessly on our production instance as is. I did mean to add this to Dynamic Filter though, not Flow Action.